viewer-prototype/src/browser/trace-viewer/components/timegraph-output-component.tsx

```diff
@@ -305,29 +320,31 @@ export class TimegraphOutputComponent extends AbstractTreeOutputComponent<Timegr
         const val = element.label;
         const modelData = element.data;
         if (modelData) {
-            const value = modelData.stateValue;
-            const elementStyle = styles[value.toString()];
-            if (elementStyle) {
+            const outputStyle = modelData.style;
+            if (!outputStyle) {
                 return {
-                    color: parseInt(elementStyle.color, 16),
-                    height: this.props.style.rowHeight * elementStyle.height,
+                    color: 0xCACACA,
```

Comment:
Ok, so we check if there's no output style instead?

Also... should we extract all the colors as constants (separate patch)

Reply:
One of my next task will be https://github.com/theia-ide/theia-trace-extension/issues/35

I'll look into the default values and everything then. I just made it work for now.